buildkite/resource_pipeline.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/shurcooL/graphql"
 )
 
+const DefaultSteps = `steps:
```

Comment:
I think this const shouldnt be exported from the package. Should we rename it to `defaultSteps`?

Reply:
That is fair - will adjust